flaskapp/core/ivr_core.py

```diff
@@ -373,9 +376,21 @@ def save_data(col_name, value, phone_number, date=None):
     all_data = gs_users_existing.get_all_records()
     if all_data:
         all_data = np.array(all_data)
-        phone_num_index = np.argmax(all_data[:, 0] == phone_number)
-        col_name_index = np.argmax(all_data[0, :] == col_name)
-        gs_users_existing.update_cell(phone_num_index, col_name_index, value)
+        phone_num_index = 1
+        for data in all_data:
+            phone_num_index = phone_num_index + 1
+            print(type(data))
```

Comment:
@Kasturi786 that's great! Quick question, do we need print here? Or this is only for debugging process?